selfdrive/car/__init__.py

```diff
@@ -132,6 +132,21 @@ def apply_std_steer_angle_limits(apply_angle, apply_angle_last, v_ego, LIMITS):
   return clip(apply_angle, apply_angle_last - angle_rate_lim, apply_angle_last + angle_rate_lim)
 
 
+def common_fault_avoidance(measured_value, max_value, request, current_request_frames=0, max_request_frames=1, cut_request_frames=1):
```

Comment:
good idea to type annotate when using generic names

Reply:
done